apache/trafficcontrol
# LICENSE file percent coverage GitHub Action

## What does this PR (Pull Request) do?

Adds a GitHub Actions workflow to ensure that [our Godocs](https://pkg.go.dev/github.com/apache/trafficcontrol/lib/go-tc) will still show up after the LICENSE file is changed by checking whether the LICENSE file is at least 80% OSS licenses, according to [pkg.go.dev](https://pkg.go.dev/).


## Which Traffic Control components are affected by this PR?


- Documentation
- Traffic Ops (technically changed, but  only comments)
- CI tests

## What is the best way to verify this PR?

* Check the result of the *License File Coverage Check for pkg.go.dev* workflow
* Build and look at the docs

## The following criteria are ALL met by this PR
